.github/workflows/monitoring-build-push-ecr.yml

```diff
@@ -0,0 +1,51 @@
+name: "Build and push on-chain monitor image to ECR"
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - monitoring/**
+
+jobs:
+  build-and-publish-monitoring:
+    runs-on: ubuntu-latest
+    environment: publish
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
+        with:
+          role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
+          role-duration-seconds: ${{ secrets.AWS_ROLE_DURATION_SECONDS }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Login to ECR
+        id: ecr
+        uses: elgohr/ecr-login-action@eb56fdd99f572d1bbb1b85cac67b98e771d53e35 # v1.0.1
+        with:
+          access_key: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret_access_key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+          region: ${{ secrets.AWS_REGION }}
+
+      - name: Output Docker Image Tag
+        id: docker_meta
+        run: |
+          GITHEAD=`git rev-parse HEAD`
+          echo ::set-output name=tag::${GITHEAD}
```

Comment:
`set-output` is deprecated and will be removed by github next year. Modern github sets `GIT_TAG_NAME` so you can just use that

Reply:
done, thanks!